src/main.rs

```diff
@@ -576,6 +578,8 @@ async fn uuid_kill_get(
         .ok_or_else(redirect::home)?
         .clone();
 
+    dbg!(&exec.lock().await.user, &session.user);
```

Comment:
Either use a logger with correct verbosity level set and more context as to what's being logged here or remove.
We don't want release builds to spam user names/IDs to the log

Reply:
yea didnt check my code had to run to check something. will remove it now